parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -272,11 +278,18 @@ ensureRunc() {
     # if on a vhd-built image, first check if we've cached the deb file
     if [ -f $VHD_LOGS_FILEPATH ]; then
         RUNC_DEB_PATTERN="moby-runc_*.deb"
-        RUNC_DEB_FILE=$(find ${RUNC_DOWNLOADS_DIR} -type f -iname "${RUNC_DEB_PATTERN}" | sort -V | tail -n1)
-        if [[ -f "${RUNC_DEB_FILE}" ]]; then
+        RUNC_DEB_FILES=$(find "${RUNC_DOWNLOADS_DIR}" -type f -iname "${RUNC_DEB_PATTERN}" 2>/dev/null || true)
```

Comment:
wouldn't `|| ""` make more sense here?

Reply:
oh typo here. But I need to do some changes here. Let me follow up.

Reply:
updated